bridge/cmd/start.go

```diff
@@ -156,6 +158,12 @@ func GetStartCmd() *cobra.Command {
 	if err := viper.BindPFlag("only", startCmd.Flags().Lookup("only")); err != nil {
 		logger.Error("GetStartCmd | BindPFlag | only", "Error", err)
 	}
+
+	startCmd.Flags().String(cli.HomeFlag, app.DefaultNodeHome, "node's home directory")
+	if err := viper.BindPFlag(cli.HomeFlag, startCmd.Flags().Lookup(cli.HomeFlag)); err != nil {
```

Comment:
Let's use cmd Flags instead of Viper. We have changed it in other modules as well.